uint8 CMD_LIST_ALL = 1
uint8 CMD_LIST_GROUP_DEF = 2 
uint8 CMD_GET_GROUP_DEF = 3
uint8 CMD_LIST_PARAM_DEF = 4
uint8 CMD_GET_PARAM_DEF = 5

uint8 cmd
uint8 group_id
uint8 param_id

